CONTRIBUTING.adoc

```diff
@@ -6,45 +6,32 @@ Thanks for your interest in this project, feel free to ask any questions you may
 
 == REQUIREMENTS
 
-To build the project, just ensure to have access to the correct golang version then just run `make build` which should return something like:
-
-```
-echo v0.0.15-6-g1e24f1d
-v0.0.15-6-g1e24f1d
-go build \
-	-ldflags "-w -s \
-        -X \"github.com/updatecli/updatecli/pkg/version.BuildTime=`date -R`\" \
-        -X \"github.com/updatecli/updatecli/pkg/version.GoVersion=go version go1.16.2 linux/amd64\" \
-        -X \"github.com/updatecli/updatecli/pkg/version.Version=v0.0.15-6-g1e24f1d\""\
-        -o bin/updatecli
-```
-
-or using docker
+To build the project, just ensure to have access to the correct golang version and link:https://goreleaser.com/install/[goreleaser installed], then just run `make build`, or use Docker:
```

Comment:
```suggestion
To build the project, make sure that you link:golang[https://go.dev/] installed, link:https://www.gnu.org/software/make/[GNU Make] and link:https://goreleaser.com/install/[goreleaser installed], then execute the command `make build`, or use Docker with:
```
- Remove occurences of `just` (even though the litteral meaning is "only", it sometimes have a deameaning sense for beginners.
- Add reference to GNU make
-  Add link to golang website